core/new-gui/src/app/workspace/types/operator-schema.interface.ts

```diff
@@ -31,6 +31,7 @@ export interface OperatorAdditionalMetadata
     dynamicInputPorts?: boolean;
     dynamicOutputPorts?: boolean;
     supportReconfiguration?: boolean;
+    allowPortCustomization?: boolean;
```

Comment:
I think if a port is "dynamically" created, then it should be allowed to configure its properties. maybe this flag can be combined with `dynamicInputPorts`?